src/nvim/move.c

```diff
@@ -2166,6 +2166,9 @@ void do_check_cursorbind(void)
         int restart_edit_save = restart_edit;
         restart_edit = true;
         check_cursor();
+        if (curwin->w_p_cul || curwin->w_p_cuc) {
```

Comment:
This is different than upstream's 8.0.0181.  It looks like you've combined that patch with part of 8.0.0188.

This PR should probably contain 8.0.0{181,182,188}, since those are all related to the same fix.